components/prf/hmac-sha256/src/prf.rs

```diff
@@ -199,17 +240,17 @@ where
             .await?;
 
         let vd = if handshake_hash.is_some() {
-            let mut outputs = self.thread_1.decode_private(&[sf_vd.vd]).await?;
+            let mut outputs = self.thread_0.decode(&[sf_vd.vd]).await?;
```

Comment:
as above, we need to make sure that `compute_server_finished_vd_private/blind` is now called `compute_server_finished_vd`.